routes/game.rb

```diff
@@ -80,9 +80,11 @@ class Api
 
                 game.save
               else
+                players = users.map { |u| [u.id, u.name] }.to_h
                 engine = Engine::GAMES_BY_TITLE[game.title].new(
-                  users.map(&:name),
+                  players,
                   id: game.id,
+                  settings: game.settings,
```

Comment:
This line is still causing an error for me when taking an action

![Screenshot from 2020-11-15 11-47-06](https://user-images.githubusercontent.com/1045173/99193754-50ab7d80-2738-11eb-9ec9-0388d6cd3c9c.png)

Reply:
Thanks for the catch